src/Login/Login.test.js

```diff
@@ -29,31 +28,24 @@ describe('Login', () => {
     expect(screen.getByText('Please type your email and password to sign in')).toBeInTheDocument()
   })
 
-  it('should welcome the user after submitting login info', async() => {
+  it('should welcome the user after submitting login info', async () => {
```

Comment:
Great fix here, looks like we were on the right track but need to make sure that we are mocking all functions before we render any components.